app/konnect/getting-started/configure-service.md

```diff
@@ -19,30 +19,69 @@ requester as responses.
 If you're following the {{site.konnect_short_name}} quickstart guide,
 make sure you have [configured a runtime](/konnect/getting-started/configure-runtime).
 
-## Add a Service and Version
+## Create a Service
 
-1. From the left navigation menu, click **Services** to open ServiceHub.
+1. From the left navigation menu, click **Services** to open Service Hub.
 
-2. Click **Add New Service**.
+1. Click **Add New Service**.
 
-3. Enter a **Service Name**. For this example, enter `example_service`.
+1. Enter a **Service Name**. For this example, enter `example_service`.
 
     A Service name can be any string containing letters, numbers, or characters;
     for example, `service_name`, `Service Name`, or `Service-name`.
 
-4. Enter a **Version Name**. For this example, enter `v.1`.
+1. (Optional) Enter a **Description**.
+
+    This description is used in Konnect only. It won't appear on the Dev
+    Portal.
+
+1. Click **Create**.
+
+    A new Service is created and the page automatically redirects back to the
+    **example_service** overview page.
+
+## Create a Service version
+
+Let's set up the first version of your API service. In Konnect, a Service can
+contain many versions, but one spec is always linked to one version.
+
+1. On your Service's overview page, scroll down to **Versions** and
+ click **New Version**.
+
+1. Enter a **Version Name**. For this example, enter `v.1`.
 
     A version name can be any string containing letters, numbers, or characters;
     for example, `1.0.0`, `v.1`, or `version#1`. A Service can have multiple
     versions.
 
-5. (Optional) Enter a **Description**.
+1. Select a runtime group.
 
-6. Click **Create**.
+    You can choose the `default` group to make this version available to all
+    users, or if you have one, select a custom group to limit this version to
+    a specific group of runtime instances. This determines which entities and
+    runtimes the Service version has access to, and who has access to this
+    version. _[link off to more info; some of this info should probably be in an intro to this guide]_
```

Comment:
Agree, runtime groups might need a separate introduction or we should intentionally skip this explanation during the getting started guide.

Reply:
I'll shorten & simplify this section for the getting started guide. The longer explanation is in this PR now: https://github.com/Kong/docs.konghq.com/pull/3770